browser/browser_context_options_test.go

```diff
@@ -62,3 +62,46 @@ func TestBrowserContextDefaultOptions(t *testing.T) {
 	assert.NotEqual(t, defaults.DeviceScaleFactor, opts.DeviceScaleFactor)
 	assert.Equal(t, defaults.Locale, opts.Locale) // should remain as default
 }
+
+func TestBrowserContextAllOptions(t *testing.T) {
+	vu := k6test.NewVU(t)
+
+	opts := common.BrowserContextOptions{
```

Comment:
What about converting a js string representation of the options to the go object? This will ensure that we've correctly mapped the fields, and the values also map correctly to the types.

```js
{
    acceptDownloads: true,
    downloadPaths: '/tmp',
    colorScheme: 'dark',
    ...
}
```

 to `common.BrowserContextOptions`.

Reply:
I believe you mean something like [this]([f449d55](https://github.com/grafana/xk6-browser/pull/1531/commits/f449d55087c23a0670394fdffc486321408eab45))?

Reply:
Yeah, perfect!